experiments/memcached-sensitivity-profile/main.go

```diff
@@ -194,9 +191,20 @@ func main() {
 						}
 					}
 
-					snapHandle, err := snapSession.LaunchSession(loadGeneratorHandle, snapTags)
+					mutilateOutput, err := loadGeneratorHandle.StdoutFile()
+					if err != nil {
+						return errors.Wrapf(err, "cannot get mutilate stdout file")
+					}
+					defer mutilateOutput.Close()
+
+					// Create snap session launcher
+					mutilateSnapSession, err := mutilatesession.NewSessionLauncherDefault(
+						mutilateOutput.Name(), snapTags)
+					errutil.CheckWithContext(err, fmt.Sprintf("Cannot create Mutilate snap session during phase %q", phaseName))
```

Comment:
@skonefal - inside iteration closure you should returned wrapped error rather than exit from the process.